Dockerfile

```diff
@@ -2,31 +2,41 @@
 FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS client
 
 # Build client
-# RUN apk add --no-cache git python g++ make
+ENV HOME_CLIENT /opt/app-root/src/app/client
+# Using root to transfer ownership of work dir
 USER root
-WORKDIR /opt/app-root/src/app/client
+RUN mkdir -p ${HOME_CLIENT}
+RUN chown -R 1001 ${HOME_CLIENT}
+USER 1001
+WORKDIR ${HOME_CLIENT}
 COPY client/package*.json ./
 RUN npm set progress=false && npm ci --no-cache
 COPY client/. .
 RUN INLINE_RUNTIME_CHUNK=false npm run build
-USER 1001
+
 # Server
 FROM registry.access.redhat.com/ubi8/nodejs-14:1 AS server
-
 # Static env vars
 ARG VERSION
 ENV VERSION $VERSION
 ENV NODE_ENV production
+ENV HOME_SEVER /opt/app-root/src/app/server
+ENV HOME_CLIENT /opt/app-root/src/app/client
```

Comment:
Is this declared twice? Once here and once on L5

Reply:
ahh it's a different container

Reply:
Yes